packages/app/cypress/e2e/index.cy.ts

```diff
@@ -647,11 +647,48 @@ describe('App: Index', () => {
           cy.findByRole('dialog', { name: defaultMessages.createSpec.successPage.header }).as('SuccessDialog').within(() => {
             cy.findByRole('link', {
               name: 'Okay, run the spec',
-            }).should('have.attr', 'href', getRunnerHref('src/App.cy.jsx')).click()
+            }).should('have.attr', 'href', getRunnerHref('cypress.config.cy.js')).click()
```

Comment:
This change is in a skipped test, is that intentional?

Reply:
it's unimportant, but yeah. i had unskipped this test to use it as the base for my other tests, and this change does need to be made. but the skipped test still fails even with this change, so i left it skipped.

Reply:
I'm going through all our TODOs/skipped tests right now, so any attempt to make them more runnable is appreciated :)